.github/sync-repo-settings.yml

```diff
@@ -0,0 +1,24 @@
+rebaseMergeAllowed: true
```

Comment:
Why are we adding this explicitly?  We really want to fall back on the common template here, no?

Reply:
Ah - I get it now.  Idea!  Even though we say it won't support node.js 10.x....  can we just keep testing it there?  Materially, nothing should prevent us from running these tests.  I think we're just trying to avoid a semver major bump in o(months), right?  This would reduce *a lot* of complexity for these new modules.

Reply:
Yep, reverted back to testing Node 10, deleting all the scaffolding around removing it...